kaybee/internal/model/policy_test.go

```diff
@@ -187,7 +187,7 @@ func TestSoftPolicy(t *testing.T) {
 	mergedStatement, mergeLog, err := s.Reduce(statementsToMerge)
 
 	fmt.Println("Merge log:")
-	for _, logEntry := range mergeLog.Entries() {
+	for _, logEntry := range mergeLog.Entries {
```

Comment:
The implementation of MergeLog is a sketchy proof-of-concept: I am not sure that yaml is a good choice (probably an embedded, searchable database is better for our scenario); for the same reason, I would prefer to keep the log entries fully encapsulated and hidden behind the "getter" `Entries()`

Reply:
I think in that case the`MergeLog` could maybe an Sqlite which would allow for searching.

Reply:
Totally agree, taht is what I had in mind. For the time being anyway, it does not matter too much which format we chose: even a file based index would do (one file for the index, and then a file for each "resolution").